cdap-ui/app/cdap/components/Reports/ReportsList/index.js

```diff
@@ -93,11 +96,16 @@ class ReportsListView extends Component {
       <div className="grid-body">
         {
           this.props.reports.map((report) => {
+            let Tag = report.status === 'RUNNING' ? 'div' : Link;
+
             return (
-              <Link
+              <Tag
```

Comment:
nit: Might be better off with two different returns based on status. React might complain that `div` doesn't have a `to` attribute defined in the dev build. Not sure if it is still the case in react 16.